llama_index/llms/huggingface.py

```diff
@@ -471,6 +471,10 @@ def class_name(cls) -> str:
             " model_name is left as default of None."
         ),
     )
+    completion_prompt_formatter: Callable[
```

Comment:
actually, each LLM already has the `completion_to_prompt` and `messages_to_prompt` kwarg in the base `LLM` class

We can pass this into the constructor kwargs and use this `complete()` and `chat()` respectively 

```
llm = HuggingFaceInferenceAPI(..., completion_to_prompt=MyFN1, messages_to_prompt=MyFN2)

<inside chat()>
prompt = self.messages_to_prompt(messages)
....
<inside complete()>
if not formatted:
    prompt = self.completion_to_prompt(completion)
```